src/index.html

```diff
@@ -12,5 +12,23 @@
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <div class="search-container">
+      <div class="search search_big" data-qa="big">
```

Comment:
```suggestion
      <div class="search search--big" data-qa="big">
```
По БЭМу это будет так